data/hydraulic-data.py

```diff
@@ -0,0 +1,147 @@
+                                                  pd.DataFrame, typing.List[pd.DataFrame]]:
+
+        def mean_conversion(pd_frame: pd.DataFrame) -> pd.Series:
+            """Means are computed along columns to average the cycle sensor data
+                
+            Each row in frame is data collected over a cycle of 60 seconds
+
+            Keyword arguments:
+            pd_frame -- pandas dataframe representation of hydraulic system sensor data
+            """
+            return pd_frame.mean(axis = 1) # axis=1 means along column
+
+        def convert_to_frame(file_bytes_object: bytes) -> pd.DataFrame:
+            """Convert bytes data to pandas dataframe
+            
+            Keyword arguments:
+            file_bytes_object -- bytes representation of hydraulic system sensor file data read using zip
+            """
+            data_file_str = str(file_bytes_object,"utf-8")
+            dt_frame = dt.Frame(data_file_str)
+            pd_frame = dt_frame.to_pandas()
+            return pd_frame
+
+        def import_data_field(zip: zipfile.ZipFile, filename: str, fieldname: str) -> pd.DataFrame:
+            """Loads pandas dataframe with average cycle sensor data
+            
+            Keyword arguments:
+            zip -- zip file object for reading filename
+            filename -- name of file to read hydraulic system sensor data from
+            fieldname -- name of hydraulic system sensor
+            """
+            pd_frame = pd.DataFrame( mean_conversion(convert_to_frame(zip.read(filename))) )
+            pd_frame.columns = [fieldname]
+            return pd_frame
+
+        def import_target_label(zip: zipfile.ZipFile, filename: str, target_label: str) -> pd.DataFrame:
+            """Loads pandas dataframe with target label field
+            
+            Keyword arguments:
+            zip -- zip file object for reading filename
+            filename -- name of file to read prediction labels from
+            target_label -- chosen hydraulic system condition monitoring label for supervised learning
+            """
+            pd_frame = convert_to_frame(zip.read(filename))
+            pd_target_labels_y = { 
+                # store hydraulic cooler condition label data from column 0 into pd frame
+                "cool_cond_y": pd.DataFrame(pd_frame.iloc[:,0]),
+                # store hydraulic valve condition label data from column 1 into pd frame
+                "valve_cond_y": pd.DataFrame(pd_frame.iloc[:,1]),
+                # store hydraulic internal pump leakage label data from column 2 into pd frame
+                "pump_leak_y": pd.DataFrame(pd_frame.iloc[:,2]),
+                # store hydraulic accumulator gas leakage label data from column 3 into pd frame
+                "acc_gas_leak_y": pd.DataFrame(pd_frame.iloc[:,3]),
+                # store hydraulic stable flag label data from column 4 into pd frame
```

Comment:
very much just a nit/observation:  Comments make up 50% of this block.  I wonder if the could be combined into a single multi-line comment at the top, perhaps making the block flow more easily w/o being separated by comments each line.